cmd/notation/cert/show.go

```diff
@@ -0,0 +1,78 @@
+package cert
+
+import (
+	"errors"
+	"fmt"
+
+	corex509 "github.com/notaryproject/notation-core-go/x509"
+	"github.com/notaryproject/notation-go/dir"
+	"github.com/notaryproject/notation/cmd/notation/internal/truststore"
+	"github.com/spf13/cobra"
+)
+
+type certShowOpts struct {
+	storeType  string
+	namedStore string
+	cert       string
+}
+
+func certShowCommand(opts *certShowOpts) *cobra.Command {
+	if opts == nil {
+		opts = &certShowOpts{}
+	}
+	command := &cobra.Command{
+		Use:   "show --type <type> --store <name> [flags] <cert_fileName>",
+		Short: "Show certificate details given trust store type, named store, and certificate file name. If the certificate file contains multiple certificates, then all certificates are displayed.",
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				return errors.New("missing certificate path")
+			}
+			if len(args) > 1 {
+				return errors.New("show only supports single certificate file")
+			}
```

Comment:
user can still hit this condition if they specify more than 2 cert files

Reply:
> user can still hit this condition if they specify more than 2 cert files

I'm not sure what do you mean here? By design, `notation cert show` only supports single cert file argument.